README.md

```diff
@@ -59,6 +59,16 @@ than mock (above).
 
     obal scratch PACKAGE
 
+However, `scratch` will not work for nightly packages (i.e. - packages that
```

Comment:
This must work for our PR testing to work.

Reply:
hrmmmmmmm. i need to think about this.